app/src/gql/apollo.js

```diff
@@ -15,13 +15,34 @@ import { split } from "apollo-link"
 import { getMainDefinition } from "apollo-utilities"
 import VueApollo from "vue-apollo"
 import { getGraphqlHost } from "scripts/url"
-import * as Sentry from "@sentry/browser"
 import config from "src/../config"
 import introspectionQueryResultData from "src/../fragmentTypes.json"
 import { getFingerprint } from "scripts/fingerprint"
 
 Vue.use(VueApollo)
 
+// to avoid circular dependencies we use this error handler, for errors happening in the router
+// those errors should trigger things in the store (that needs the router)
```

Comment:
where is this error handler communicating with store? I don't see it

Reply:
Another question: why we don't capture these router errors with Sentry anymore? They are not so critical?